packages/@azure/servicebus/data-plane/lib/util/utils.ts

```diff
@@ -78,7 +78,7 @@ function reorderLockToken(lockToken: string): Buffer {
  * @param lockTokens An array of lock tokens whose bytes need to be reorderd.
  * @returns Buffer[] An array of Buffer representing reordered bytes.
  */
-export function reorderLockTokens(lockTokens: string[]): Buffer[] {
+export function reorderLockTokens(lockTokens: Buffer[]): Buffer[] {
```

Comment:
I don't understand why we need `reorderLockTokens`. From what I can see, we only ever deal with 1 lock token at a time. So maybe its better to remove `reorderLockTokens` and export just`reorderLockToken`?

It will make the code cleaner in `serviceBusMessage.ts` file where we have to call `reorderLockTokens` and then use the zero index to fetch the first entry in the array

Reply:
Ok, I'll remove extra `reorderLockTokens` method.